examples/heartbeat_demo/alicia.py

```diff
@@ -0,0 +1,142 @@
+from nucypher.characters.lawful import Bob, Ursula
+from nucypher.config.characters import AliceConfiguration
+from nucypher.config.storages import LocalFileBasedNodeStorage
+from nucypher.crypto.powers import EncryptingPower, SigningPower
+from nucypher.network.middleware import RestMiddleware
+from nucypher.utilities.logging import simpleObserver
+from umbral.keys import UmbralPublicKey
+
+from doctor_keys import get_doctor_pubkeys
+
+import datetime
+import os
+import shutil
+import maya
+import json
+from twisted.logger import globalLogPublisher
+
+
+POLICY_FILENAME = "policy.json"
+
+
+######################
+# Boring setup stuff #
+######################
+#
+# # Twisted Logger
+globalLogPublisher.addObserver(simpleObserver)
```

Comment:
Hmm - Ins't `simpleObserver` a class now?

```
globalLogPublisher.addObserver(SimpleObserver())
```

Reply:
Yeah, there are many things that are outdated with respect to current `master`, but I wanted something that could be tested on testnet. I'll update everything when testnet is again compatible with `master`.

Reply:
Ah - Got it - didn't notice this is based on federated.